src/server/validation/validateRequest.ts

```diff
@@ -16,6 +16,11 @@ export interface ReqValidationOptions<ERR extends Error> {
    * If true - `genericErrorHandler` will report it to errorReporter (aka Sentry).
    */
   report?: boolean | ((err: ERR) => boolean)
+
+  /**
```

Comment:
p3: I have a habit to explicitly document what is a default value of every option, e.g here I'd write like `Defaults to false, which means it would **convert**/mutate the original value. Set to true to force non-mutating behavior`.

Your description is good, but I like the emphasis on "what is default, and what happens when you flip the default".